### PR TITLE
Add Testing Ghost-Roles

### PR DESCRIPTION
**Pull request by Ultradragon005**

### CODE REVIEW

**Comment by Tommy-XL** [comment]
You mean add ghost roles in `/up` command?

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -3614,5 +3614,6 @@
   "WardenIncreaseSpeed": "Increase Speed By",
   "WardenWarn": "DANGER! RUN!",
 
-  "MinionAbilityTime": "Ability Duration"
+  "MinionAbilityTime": "Ability Duration",
+  "blind": "blinded"
```
Comment (Tommy-XL) [inline Resources/Lang/en_US.json:3618]: Can you make `Minion_Blind`

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:3618]: I mean sure I'll do it rn, but the reason I simply called it "blind" is hence it is a generic attribute which could potentially have use in the future for smt else (smt like what happened with arrogance/reverie)

**Comment by Ultradragon005** [comment]
> You mean add ghost roles in `/up` command?

I mean I could do that, currently it's only set for dev since I wanna be able to test them in people's lobbies.

**Comment by Ultradragon005** [comment]
should I tho, idk.. kinda ruins the point of a ghost-role IMO

**Comment by Tommy-XL** [comment]
> I mean I could do that, currently it's only set for dev since I wanna be able to test them in people's lobbies.

Ok

**Comment by Ultradragon005** [comment]
> Ok

wdym

**Pull request merged**